lib/apiservers/portlayer/restapi/handlers/misc_handlers.go

```diff
@@ -16,30 +16,40 @@ package handlers
 
 import (
 	"github.com/go-swagger/go-swagger/httpkit/middleware"
+	"golang.org/x/net/context"
 
 	"github.com/vmware/vic/lib/apiservers/portlayer/models"
 	"github.com/vmware/vic/lib/apiservers/portlayer/restapi/operations"
 	"github.com/vmware/vic/lib/apiservers/portlayer/restapi/operations/misc"
 	"github.com/vmware/vic/lib/portlayer/exec"
 )
 
-type MiscHandlersImpl struct{}
+// MiscHandlersImpl is the receiver for all the misc handler methods
+type MiscHandlersImpl struct {
+	ctx *context.Context
+}
 
 // Configure assigns functions to all the miscellaneous api handlers
 func (handler *MiscHandlersImpl) Configure(api *operations.PortLayerAPI, handlerCtx *HandlerContext) {
 	api.MiscPingHandler = misc.PingHandlerFunc(handler.Ping)
 	api.MiscGetVCHInfoHandler = misc.GetVCHInfoHandlerFunc(handler.GetVCHInfo)
+
+	handler.ctx = handlerCtx.Context
 }
 
 // Ping sends an OK response to let the client know the server is up
 func (handler *MiscHandlersImpl) Ping() middleware.Responder {
 	return misc.NewPingOK().WithPayload("OK")
 }
 
+// GetVCHInfo returns VCH-related info for a `docker info` call
 func (handler *MiscHandlersImpl) GetVCHInfo() middleware.Responder {
+	vchCPUMhz := exec.NCPU(*handler.ctx)
```

Comment:
Please create a new context here: `context.Background()` instead of storing one in the HandlerContext.
Storing a context in that manner, where it will be used across multiple distinct operations is not good practice.